<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNavigtion</class>
 <widget class="QWidget" name="CNavigtion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNavigtion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CNavigtion{background:rgb(0,58,108,150);border-radius:4px;border:1px solid gray}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
     <item>
      <widget class="QWidget" name="widgetLeft" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#widgetLeft{background:rgb(255,255,255,255);border-radius:4px;border:1px solid gray}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widgetAllItem" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetItem" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <property name="margin">
                 <number>2</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QWidget" name="widgetItems" native="true">
                  <layout class="QGridLayout" name="gridLayout_7">
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="vLayoutFun"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTreeWidget" name="treeWidgetItems">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="headerHidden">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFunSwitch">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">/*登录按钮样式*/
QPushButton {
	border-radius:1px;
	border:none;
	background-color: #ffffff;
	color:#000000;
	text-align: left;
	height:22px;
	padding-left:6px;
}

/*登录按钮鼠标悬停样式*/
QPushButton:hover {
	background-color:rgb(194,220,252);
	border:1px solid rgb(21,131,221)
}</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widgetFind" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEditFind">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border:1px solid rgb(209,209,209);
	height:22px;
	font-size:10pt;
	padding-left:6px;
	padding-right:6px;
	background-image:url(:/images/search);
	background-repeat:no;
	background-position:right center;
	background-origin: content;
}

QLineEdit:hover {
	border:1px solid rgb(21,131,221)
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
       <zorder>widgetFind</zorder>
       <zorder>widgetAllItem</zorder>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelUser">
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUser">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="styleSheet">
          <string notr="true">color: gray;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlugin">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>插件管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUsers">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>用户权限</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFunPoint">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>功能点管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="styleSheet">
          <string notr="true">color: gray;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSysConfig">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>系统配置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLog">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>日志配置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDB">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>数据库配置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="styleSheet">
          <string notr="true">color: gray;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHelp">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>帮助及支持</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnUserSwitch">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>切换用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuit">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius:1px;
	border:none;
	background: transparent;
	color:#ffffff;
	text-align: left;
	height:35px;
	padding-left:10px;
}

QPushButton::hover {
	border-image:url(:/images/btn_bg)
}</string>
         </property>
         <property name="text">
          <string>退出系统</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skgui.qrc"/>
 </resources>
 <connections/>
</ui>
